.github/workflows/integration.yml

```diff
@@ -175,14 +175,16 @@ jobs:
           DIRECTORY: '${{ matrix.working_directory }}'
           # used to create comments on pull requests (access to only this repo)
           REPO_TOKEN: '${{ github.token }}'
+          JOB_NAME: '${{ github.job }} (${{ matrix.working_directory }})'
```

Comment:
I would also add a `name: 'plan (${{ matrix.working_directory }})'` to the job also. Above line 126 (for both apply and plan workflows)

Reply:
Done